File: 06. Perulangan/E. Dua Pangkat/e_dua_pangkat.pas
program e_dua_pangkat;
var
  N : longint;
begin
  readln(N);
  while (N mod 2 = 0) do
    N := N div 2;
  if (N = 1) then
    writeln('ya')
  else
    writeln('bukan')
end.

File: 06. Perulangan/E. Dua Pangkat/e_dua_pangkat.cpp
#include <iostream>
int main(){
  int N;
  std::cin >> N;
  while (N % 2 == 0)
    N /= 2;
  N == 1 ? std::cout << "ya" << std::endl : std::cout << "bukan" << std::endl;
  return 0;
}
